core/object.cpp

```diff
@@ -1073,23 +1073,21 @@ Error Object::emit_signal(const StringName &p_name, const Variant **p_args, int
 	//copy on write will ensure that disconnecting the signal or even deleting the object will not affect the signal calling.
 	//this happens automatically and will not change the performance of calling.
 	//awesome, isn't it?
-	VMap<Signal::Target, Signal::Slot> slot_map = s->slot_map;
-
-	int ssize = slot_map.size();
+	OrderedHashMap<Signal::Target, Signal::Slot, Signal::TargetHasher> slot_map = s->slot_map;
```

Comment:
I wrote the comment above for a reason, and you are ignoring it. This will break.